cmd/kube-controller-manager/app/core.go

```diff
@@ -346,6 +347,25 @@ func startNamespaceController(ctx ControllerContext) (http.Handler, bool, error)
 	return startModifiedNamespaceController(ctx, namespaceKubeClient, nsKubeconfigs)
 }
 
+func startTenantController(ctx ControllerContext) (http.Handler, bool, error) {
+	// tenant controller will be even more chatty than namespace controller when doing cleanups.
+	tnKubeConfigs := ctx.ClientBuilder.ConfigOrDie("tenant-controller")
```

Comment:
It is mentioned that a tenant controller is likely to be more chatty than a namespace-controller, do we need to enlarge the value of QPS or Burst?

If making a decision is unrealistic now, I would like some comments here for future code maintainer.

Reply:
It's mostly chatty during tenant cleanup because it may need to delete lots of resources. But I'm not sure if we really need to increase QPS or burst, because deleting a tenant doesn't have to be very quickly.  Anyway, I think I can update comments for later consideration and evaluation.